src/layer/__tests__/fixtures.outputResponses.ts

```diff
@@ -0,0 +1,48 @@
+import { S2L2ALayer, BBox, CRS_EPSG4326, MimeTypes } from '../../index';
```

Comment:
IIUC, we don't actually need this fixture at all, because in the tests we don't care about the responses (we don't actually test the result), we just check that the correct request was made.

I would suggest removing it for clarity (the next person reading this will need to understand the same thing), but it's not a big deal. Fftr.

Reply:
Thanks, yes, we don't care about responses (well, not yet, maybe we should add such test in the future). 
Removed this file and moved the needed things (`layer` and `getMapParams`) to `outputResponses.ts`.
Will resolve